testpilot/frontend/static-src/test/views/experiment-tour-dialog-view.js

```diff
@@ -0,0 +1,39 @@
+import test from 'tape-catch';
+import Experiment from '../../app/models/experiment';
+import View from '../../app/views/experiment-tour-dialog-view';
+
+test(`Running Tests for ${__filename}`, a => a.end());
+
+test('Experiment tour dialog view renders', t => {
+  t.plan(1);
+
+  const model = new Experiment({
+    slug: 'test',
+    title: 'test',
+    tour_steps: [
+      {copy: 'step 1', image: '1.gif'},
+      {copy: 'step 2', image: '2.gif'},
+      {copy: 'step 3', image: '3.gif'}
+    ]
+  });
+
+  const view = new View({
+    model: model
+  });
+
+  view.render();
+
+  t.ok(view.query('.onboarding-modal'));
+
+  // const content = view.queryAll('.tour-content');
```

Comment:
Yeah, this is the test I mentioned that's not working yet and I'm still poking at